src/SmoFacade/BackupFileTools.cs

```diff
@@ -64,5 +64,17 @@ public static bool IsValidFilePath(string path)
         return false;
       }
     }
+
+    public static string CombinePaths(string path1, string path2)
+    {
+      if (string.IsNullOrEmpty(path1)) { return path2; }
+      if (string.IsNullOrEmpty(path2)) { return path1; }
```

Comment:
Should we consider the cases where one or both paths are invalid? 

For example: 
`CombinePaths("", "C:/is/not/\valid")` 
`CombinePaths("C:\\dir/", "\/file.ext")` 

It's probably unlikely to end up in that state but it'd be nice to check if the path is valid before returning anything.

Reply:
If it's not done already you can pass the inputs through:
`IsValidFilePath()`

https://github.com/factset/AgDatabaseMove/blob/47aaae40d3da0269679504590e1a7b22a411083d/src/SmoFacade/BackupFileTools.cs#L52-L67

Reply:
Might be overkill, depends on where these inputs are coming from

Reply:
I'm not opposed to adding a couple calls to `IsValidFilePath`, but the method should be refactored away from the use of `Path.*` first. That should be done in a subsequent change.